gatsby-theme-landing-page/src/styles/variables.module.css

```diff
@@ -5,8 +5,8 @@
  */
 .root {
   /* typography */
-  --font: system-ui, sans-serif;
```

Comment:
I think these demo-specific styles ought to live in the starter, whereas the theme's styles should be fairly normal/boring. You should be able to lift these changes into `src/gatsby-theme-landing-page/styles/variables.module.css` to do this. This would also be a good opportunity to ensure the styles/layout/etc can be properly customized from the starter and we can round out any potential issues with the shadowing API

Reply:
Sounds good. Which styles are you considering demo-specific?

Reply:
I think pretty much everything in this change looks like it's for the demo; I'd also recommend we use blue for hover states